tests/rip7560/test_send_failed.py

```diff
@@ -162,7 +162,7 @@ def test_eth_send_paymaster_validation_calls_invalid_callback(
     response = tx_7560.send()
     assert_rpc_error(
         response,
-        "paymaster validation did call the EntryPoint but not the 'acceptPaymaster' callback",
+        "validation phase failed with exception: unable to decode acceptPaymaster: no method with id: 0x9a0e28f8",
```

Comment:
This is an extremely weird error message. What was wrong with the old one?

Reply:
the scenario: paymaster called entrypoint with wrong callback. what is the suggested error message?
Currently, there is no special handling, so it reports the methodsig (= acceptPaymasterWrongSig). Do you have a better error message ? I don't want to say "didn't call", since it did call  something (just to the wrong method/params)